src/index.css

```diff
@@ -0,0 +1,19 @@
+@tailwind base;
+@tailwind components;
+@tailwind utilities;
+
+body {
+  color: #ffefef;
+  background-color: teal;
+}
+
```

Comment:
- [ ] As per this [requirement](https://github.com/microverseinc/curriculum-react-redux/blob/main/capstone/react_capstone.md#:~:text=Typography%3A%20font%20face%2C%20size%20and%20weight.), Please add the correct font families for all text elements in your design. Kindly use `gill sans` as the body and title font and `lato` as the control font.

![image](https://github.com/hamzaw9/metrics/assets/68796232/c34276a8-f7b2-464d-afe0-f3a0e5da9a7e)